### DISCUSSION TITLE
How to make a Union with Enum?

### DISCUSSION THREAD

**Discussion by henriqueArrazao** [MichalLytek/type-graphql#1223] [Q&A]
![image](https://user-images.githubusercontent.com/65516006/156897247-3c504752-16a6-4d1c-bd30-eca25e3bd0df.png)

I can create a class an object to wrap the enum if there is no other alternative

**Comment by MichalLytek** [comment]
Not possible by GraphQL spec

**Reply by henriqueArrazao** [reply]
Ok, thank you
